<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsWidget</class>
 <widget class="QWidget" name="StatisticsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <widget class="PickPeriodWidget" name="widgetPickPeriod" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>1011</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>971</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="4">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>daily average</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>total pomodoros</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelTotalPomodoros">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>402</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="labelDailyAverage">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>14.36</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="6">
      <widget class="Plot" name="dailyTimeline" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>969</width>
         <height>191</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>1011</width>
     <height>311</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1011</width>
     <height>311</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>311</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>1041</width>
     <height>681</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>370</y>
      <width>331</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutBestWorktime">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>.Helvetica Neue Desk UI</family>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Best Worktime</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelBestWorktimeName">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>.Helvetica Neue Desk UI</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Afternoon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelBestWorktimeHours">
       <property name="font">
        <font>
         <family>.Helvetica Neue Desk UI</family>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>14:00-18:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="DistributionDiagram" name="topTagDiagram" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>251</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>.Helvetica Neue Desk UI</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Best Workday</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelBestWorkdayName">
       <property name="font">
        <font>
         <family>.Helvetica Neue Desk UI</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>wordayNamePlaceholder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelBestWorkdayMsg">
       <property name="font">
        <font>
         <family>.Helvetica Neue Desk UI</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Average comparison message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BarChart" name="workdayBarChart" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame_3</zorder>
  <zorder>widgetPickPeriod</zorder>
  <zorder>frame</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PickPeriodWidget</class>
   <extends>QWidget</extends>
   <header>src/widgets/pickperiodwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DistributionDiagram</class>
   <extends>QWidget</extends>
   <header>src/widgets/DistributionDiagram.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BarChart</class>
   <extends>QWidget</extends>
   <header>src/widgets/barchart.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Plot</class>
   <extends>QWidget</extends>
   <header>src/widgets/plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
